### ISSUE TITLE
Missed Description for `mandatory-home` Warning

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#209]
I'm getting the following warning when I check my `xmir`:

```
[mandatory-home WARNING]:0 The +home meta is mandatory, but is absent
```

I don't have `home` meta and I don't know what I should put here. Please add more human-readable description to this warning and provide instruction that should be done to fix this warning.

There is how I check the `xmir`:

```java
final Collection < Defect > defects = new Program(new StrictXmir(this.doc)).defects()
    .stream()
    .collect(Collectors.toList());
if (!defects.isEmpty()) {
    throw new IllegalStateException(
        String.format(
            "XMIR is incorrect: %s, %n%s%n",
            defects,
            this.doc
        )
    );
}
```

Full `xmir`:
[MethodByte.xmir.txt](https://github.com/user-attachments/files/18358754/MethodByte.xmir.txt)

**Comment by volodya-lombrozo** [comment]
Blocks: https://github.com/objectionary/jeo-maven-plugin/issues/946

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you have a look, please?

**Comment by yegor256** [comment]
@h1alexbel can you please help here and provide a better error message. I believe, we should say:

```
The +home meta is mandatory, but is absent (must contain URL of repository where object sources may be found)
```

**Comment by yegor256** [comment]
@volodya-lombrozo in can of JEO, I would put there this URL (for all objects): https://github.com/objectionary/jeo-maven-plugin

**Comment by volodya-lombrozo** [comment]
> @h1alexbel can you please help here and provide a better error message. I believe, we should say:
> 
> ```
> The +home meta is mandatory, but is absent (must contain URL of repository where object sources may be found)
> ```

@yegor256 @h1alexbel  It should also contain an example of this meta, otherwise, it's impossible to say which child elements the `home` meta contains.

@yegor256 Can you provide an example, please?

**Comment by h1alexbel** [comment]
@yegor256 assign me, please

**Comment by h1alexbel** [comment]
> It should also contain an example of this meta, otherwise, it's impossible to say which child elements the home meta contains.

@yegor256 @volodya-lombrozo I suggest we should implement [this](https://github.com/objectionary/lints/issues/207#issuecomment-2579710093), in order to resolve this type of problems. For instance, for `mandatory-home` we will link HTML page with this document: https://github.com/objectionary/lints/blob/gh-pages/_site/0.0.30/mandatory-home-meta.md. WDYT?

**Comment by yegor256** [comment]
@h1alexbel if we put links to each `message` of each `Defect`, it will be pretty noisy for the eo-maven-plugin. Maybe it's better to add an explanation to the Javadoc of the `Defect`, `Program.defects()`, and `Programs.defects()`.

**Comment by h1alexbel** [comment]
@yegor256 Should we add a link to the blog post about XMIR? Or to the XMIR documentation?

**Comment by yegor256** [comment]
@h1alexbel I suggest we add both links

**Comment by h1alexbel** [comment]
@volodya-lombrozo still an issue?

**Comment by volodya-lombrozo** [comment]
@h1alexbel I'll check it soon

**Comment by volodya-lombrozo** [comment]
@h1alexbel @yegor256 It works if I add:

```xml
      <meta>
         <head>home</head>
         <tail>https://github.com/objectionary/jeo-maven-plugin</tail>
         <part>https://github.com/objectionary/jeo-maven-plugin</part>
      </meta>
```
But if I leave just:

```xml
      <meta>
         <head>home</head>
         <tail>https://github.com/objectionary/jeo-maven-plugin</tail>
      </meta>
```

It fails with the same error.
Is it necessary to duplicate URL? Moreover, it's unclear what `tail` and `part` mean.

**Comment by h1alexbel** [comment]
@volodya-lombrozo in `meta` node, `tail` and `part` are required to present meta content. While `tail` shows the full tail, `part` stands only for specific _part_ in the meta tail. Here is an example:

```xml
<meta>
   <head>home</head>
   <tail>https://github.com/objectionary/jeo-maven-plugin</tail>
   <part>https://github.com/objectionary/jeo-maven-plugin</part>
</meta>
```

In this case it means: `+meta home https://github.com/objectionary/jeo-maven-plugin` (tail with one part).

Also, you can create the meta with multiple parts: `+rt jvm org.eolang:eo-runtime:0.0.0` (one `tail` with two `part`s)

The second example is invalid, since there is a `tail`, but there is no parts.

**Comment by h1alexbel** [comment]
@volodya-lombrozo still an issue?

**Comment by volodya-lombrozo** [comment]
@h1alexbel No, thanks. Now the message is more or less clear.

```
 The +home meta is mandatory, but is absent (must contain URL of repository where object sources may be found)
```
